### PR TITLE
Release `0.19.4`

### PR DESCRIPTION
**Pull request by ElenaKhaustova**
## Description
Release 0.19.4 🎉

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -48,7 +48,9 @@ authors:
   given-names: Merel
 - family-names: Zabłocki
   given-names: Marcin
+- family-names: Khaustova
+  given-names: Elena
```
Comment (merelcht) [inline CITATION.cff:52]: This list is actually the same as our TSC list: https://github.com/kedro-org/kedro/pull/3565 (or it should be). We still need to cast a vote to add you to the TSC, which I think @astrojuanlu is doing soon 🤞 

That said, this does mean Jo Stichbury https://github.com/kedro-org/kedro/pull/3647 can be removed from this list now and Simon Brugman should be added https://github.com/kedro-org/kedro/pull/3780.

**Review by astrojuanlu — APPROVED** [review]
:shipit:

**Review by merelcht — COMMENTED** [review]
image

**Review by merelcht — APPROVED** [review]

**Review by ankatiyar — APPROVED** [review]
💯

**Review by SajidAlamQB — APPROVED** [review]
loool

**Review by lrcouto — APPROVED** [review]
That's how we all feel NGL

**Review by rashidakanchwala — APPROVED** [review]

**Comment by noklam** [comment]
I suggest this in RELEASE.md as I cannot comment on unchanged file in PR.

```md
#  Release 0.19.4

## Major features and improvements
* Kedro commands now work from any subdirectory within a Kedro project.
* Kedro CLI now provides a better error message when project commands are run outside of a project i.e. `kedro run`
* Added the `--telemetry` flag to `kedro new`, allowing the user to register consent to have user analytics collected at the same time as the project is created.
* Improved suggestions to resume failed pipeline runs.
* Improved the performance of `Pipeline` object creation and summing.
* Dropped the dependency on `toposort` in favour of the built-in `graphlib` module.
* Cookiecutter errors are shown in short format without the `--verbose` flag.

## Bug fixes and other changes
* Updated `kedro pipeline create` and `kedro pipeline delete` to read the base environment from the project settings.
* Updated CLI command `kedro catalog resolve` to read credentials properly.
* Changed the path of where pipeline tests generated with `kedro pipeline create` from `<project root>/src/tests/pipelines/<pipeline name>` to `<project root>/tests/pipelines/<pipeline name>`.
* Updated ``.gitignore`` to prevent pushing Mlflow local runs folder to a remote forge when using mlflow and git.
* Fixed error handling message for malformed yaml/json files in `OmegaConfigLoader`.
* Fixed a bug in `node`-creation allowing self-dependencies when using transcoding, that is datasets named like `name@format`.
* Improved error message when passing wrong value to node.

## Breaking changes to the API
* Methods `_is_project` and `_find_kedro_project` have been moved to `kedro.utils`. We recommend not using private methods in your code, but if you do, please update your code to use the new location.

## Documentation changes
* Added documentation on best practices for testing nodes and pipelines.
* Added missing description for `merge_strategy` argument in OmegaConfigLoader.
* Clarified docs around using custom resolvers without a full Kedro project.
```

**Review by noklam — COMMENTED** [review]
We need to update the `RELEASE.md`

**Comment by ElenaKhaustova** [comment]
> We need to update the `RELEASE.md`

Done

**Review by AhdraMeraliQB — APPROVED** [review]

**Pull request merged**